tests/ui/loops/loop-if-else-break-issue-123261.stderr

```diff
@@ -0,0 +1,59 @@
+error[E0268]: `break` outside of a loop or labeled block
+  --> $DIR/loop-if-else-break-issue-123261.rs:9:13
+   |
+LL |             break 1;
+   |             ^^^^^^^ cannot `break` outside of a loop or labeled block
+   |
+help: consider labeling this block to be able to break within it
+   |
+LL ~     let x = 'block: {
+LL |         if n == 0 {
+LL ~             break 'block 1;
+   |
+
+error[E0268]: `break` outside of a loop or labeled block
+  --> $DIR/loop-if-else-break-issue-123261.rs:11:13
+   |
+LL |             break 2;
+   |             ^^^^^^^ cannot `break` outside of a loop or labeled block
+   |
+help: consider labeling this block to be able to break within it
+   |
+LL ~ 'block: //@ run-rustfix
+LL | 
+ ...
+LL |         } else {
+LL ~             break 'block 2;
+   |
+
+error[E0268]: `break` outside of a loop or labeled block
+  --> $DIR/loop-if-else-break-issue-123261.rs:17:13
+   |
+LL |             break 1;
+   |             ^^^^^^^ cannot `break` outside of a loop or labeled block
+   |
+help: consider labeling this block to be able to break within it
+   |
+LL ~     let y = 'block: {
+LL |         if n == 0 {
+LL ~             break 'block 1;
+   |
+
+error[E0268]: `break` outside of a loop or labeled block
+  --> $DIR/loop-if-else-break-issue-123261.rs:19:9
+   |
+LL |         break 0;
+   |         ^^^^^^^ cannot `break` outside of a loop or labeled block
+   |
+help: consider labeling this block to be able to break within it
+   |
+LL ~ 'block: //@ run-rustfix
+LL | 
+ ...
+LL |         }
+LL ~         break 'block 0;
+   |
```

Comment:
This suggestion is less than ideal. We shouldn't be emitting it, instead checking for the block which is RHS of the `let`.

Reply:
> This suggestion is less than ideal. We shouldn't be emitting it, instead checking for the block which is RHS of the `let`.

Thank you, I'm not sure if I understand correctly, is the comment referring to here?
```rust
// before
let x = {
    break 0;
};

//after fix:

let x = 'b: {
    break 'b 0;
};
```

The fix suggestion for this is not introduced by this PR and it really fix the error.

Reply:
> This suggestion is less than ideal. We shouldn't be emitting it, instead checking for the block which is RHS of the `let`.

Thank you. Please see if this fix suggestion is ​​better.